exporters/otlp/src/otlp_grpc_client.cc

```diff
@@ -103,6 +103,60 @@ std::unique_ptr<grpc::ClientContext> OtlpGrpcClient::MakeClientContext(
   return context;
 }
 
+std::unique_ptr<grpc::CompletionQueue> OtlpGrpcClient::MakeCompletionQueue()
```

Comment:
where is this method being used ?

Reply:
I plan use it to implement concurrency exporting in the future. Also , the `grpc::CompletionQueue` is also used by `Export` .I think we better export it.